cedar-lean/UnitTest/IPAddr.lean

```diff
@@ -164,6 +166,6 @@ def tests := [
   testsForIpNetEquality]
 
 -- Uncomment for interactive debugging
--- #eval TestSuite.runAll tests
+#eval TestSuite.runAll tests
```

Comment:
did we intend to check-in this uncomment? if so, we can adjust the comment that says "Uncomment"

Reply:
Good catch; that was not intentional!

Fixing now.